cli_tools_tests/e2e/README.md

```diff
@@ -0,0 +1,85 @@
+e2e tests: Invoke gcloud and the wrapper binaries using their public APIs.
+
+## Overview
+
+Each directory adjacent to this README is a test suite. At the time of writing, there are four test suites:
+
+```
+gce_image_import_export
+gce_ovf_export
+gce_ovf_import
+gce_windows_upgrade
+```
+
+We build, deploy, and invoke the e2e tests using docker images, with one image per test suite. The docker image
+contains:
+
+1. Tests, as a compiled go binary
+2. One or more wrappers, as a compiled go binary.
+3. gcloud
+
+The Dockerfiles that generate these images are in the root of this repository. At the time of writing, they are:
+
+```
+gce_image_import_export_tests.Dockerfile
+gce_ovf_export_tests.Dockerfile
+gce_ovf_import_tests.Dockerfile
+gce_windows_upgrade_tests.Dockerfile
+```
+
+## Building
+
+Build e2e Dockerfiles using `docker build -f <docker file name>`.
+
+This example builds the tests associated with `gce_image_import_export_tests.Dockerfile`, and stores the image in a
+tag `gce_image_import_export_tests`.
+
+From the root of this repository, execute:
+
+```shell
+gcloud auth login
+gcloud auth configure-docker
+docker build  -f gce_image_import_export_tests.Dockerfile  . -t e2e
+```
+
+Notes:
+
+1. The first two gcloud commands ensure that your local docker installation can read
+   the `gcr.io/compute-image-tools-test/wrapper-with-gcloud:latest` image.
+
+Use the same syntax to build other e2e Dockerfiles.
+
+## Running
+
+Run e2e docker images using `docker run <tag name>`.
+
+This example runs a subset of the tests from the image built in the prior step.
+
+```shell
```

Comment:
Does it make sense to create a script that does this? Can be a small task for Nooglers.

Reply:
Sounds good -- added a script.